README.md

```diff
@@ -19,3 +18,41 @@ https://www.youtube.com/playlist?list=PLqnslRFeH2Upcrywf-u2etjdxxkL8nl7E
 - K-Means
 - AdaBoost
 - Linear Discriminant Analysis (LDA)
+
+## Installation and usage.
+
+This project has 2 dependencies.
+
+- `numpy` for the maths implementation and writing the algorithms
+- `Scikit-learn` for the data generation and testing.
+- `Matplotlib` for the plotting.
+- `Pandas` for loading data.
+
+**NOTE**: Do note that, Only `numpy` is used for the implementations. Others
+help in the testing of code, and making it easy for us, instead of writing that
+too from scratch.
+
+You can install these using the command below!
+
+```sh
+# Linux or MacOS
+pip3 install -r requirements.txt
+
+# Windows
+pip install -r requirements.txt
+```
+
+You can run the files as following.
+
+```sh
+python -m algorithms.<algorithm-file>
```

Comment:
please fix now after reverting back to old naming

Reply:
Sorry! I forgot to change this instance too 😅